From 45b575b791dbd3d5660a0c08065a9fbcb6e21eb9 Mon Sep 17 00:00:00 2001
From: Seiichi Ikarashi <s.ikarashi@jp.fujitsu.com>
Date: Wed, 10 Jun 2015 07:29:03 -0300
Subject: [PATCH 2/5] rasdaemon: remove a space from mcgstatus_msg

"ras-mc-ctl --errors" shows an unnecessary space character in the
mcgstatus string of MCE event, like below:

2 2015-04-04 19:57:22 +0900 error: MC_HA_IMC_RW_BLOCK_ACK_TIMEOUT, mcg mcgstatus= 0, mci Corrected_error, mcgcap=0x07000c16, status=0x8000000067000e0b, walltime=0x555da140, cpu=0x00000001, cpuid=0x000306f3, apicid=0x00000002, bank=0x00000004

Let's remove it.

Signed-off-by: Seiichi Ikarashi <s.ikarashi@jp.fujitsu.com>
Signed-off-by: Mauro Carvalho Chehab <mchehab@osg.samsung.com>
---
 mce-intel.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mce-intel.c b/mce-intel.c
index 3503c6a..77b929b 100644
--- a/mce-intel.c
+++ b/mce-intel.c
@@ -176,7 +176,7 @@ static void decode_mcg(struct mce_event *e)
 {
 	uint64_t mcgstatus = e->mcgstatus;
 
-	mce_snprintf(e->mcgstatus_msg, "mcgstatus= %lld",
+	mce_snprintf(e->mcgstatus_msg, "mcgstatus=%lld",
 		     (long long)e->mcgstatus);
 
 	if (mcgstatus & MCG_STATUS_RIPV)
-- 
1.8.3.1

